diff -rupN echoprint-codegen.old/src/File.h echoprint-codegen/src/File.h
--- echoprint-codegen.old/src/File.h	2012-01-14 21:54:21.000000000 -0500
+++ echoprint-codegen/src/File.h	2012-01-15 13:24:43.437071589 -0500
@@ -9,6 +9,8 @@
 #include <string.h>
 #ifdef _WIN32
 #include "win_unistd.h"
+#else
+#include <unistd.h>
 #endif
 /*
 This makes file writing a bit easier (makes sure we don't forget to fclose, basically). Use it like this:
